### PR TITLE
Adds recipe for trafaret

### PR DESCRIPTION
**Pull request by elatt**

### CODE REVIEW

File: recipes/trafaret/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2013, 2014, Mikhail Krivushin and Yehor Nazarkin.
```
Comment (CJ-Wright) [inline recipes/trafaret/LICENSE.txt]: I don't think this file is needed if the license is included in the package (we should be able to pull it from the tarball). If it isn't please follow the steps for [packaging a license](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).

**Comment by elatt** [comment]
@CJ-Wright it looks as though all checks are green, what would be the next steps for getting this merged?

File: recipes/trafaret/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Trafaret is rigid and powerful lib to work with foreign data, configs etc.
+    It provides simple way to check anything, and convert it accordingly to
+    your needs.
+  doc_url: http://trafaret.readthedocs.io/en/latest/
+  dev_url: https://github.com/Deepwalker/trafaret
+
+extra:
+  recipe-maintainers:
+    - elatt
+    - dsakagi
```
Comment (CJ-Wright) [inline recipes/trafaret/meta.yaml]: @dsakagi are you ok with being a maintainer?

File: recipes/trafaret/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - peterhurford
```
Comment (CJ-Wright) [inline recipes/trafaret/meta.yaml]: @peterhuford are you ok with being a maintainer?

File: recipes/trafaret/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    - madmott
```
Comment (CJ-Wright) [inline recipes/trafaret/meta.yaml]: @madmott are you ok with being a maintainer?

File: recipes/trafaret/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    - nott
```
Comment (CJ-Wright) [inline recipes/trafaret/meta.yaml]: @nott are you ok with being a maintainer?

File: recipes/trafaret/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    - Deepwalker
```
Comment (CJ-Wright) [inline recipes/trafaret/meta.yaml]: @Deepwalker are you ok with being a maintainer?

File: recipes/trafaret/meta.yaml (hunk 4 above)
Comment (madmott) [reply recipes/trafaret/meta.yaml]: yes

File: recipes/trafaret/meta.yaml (hunk 3 above)
Comment (peterhurford) [reply recipes/trafaret/meta.yaml]: Yes

File: recipes/trafaret/meta.yaml (hunk 6 above)
Comment (Deepwalker) [reply recipes/trafaret/meta.yaml]: yes

File: recipes/trafaret/meta.yaml (hunk 5 above)
Comment (nott) [reply recipes/trafaret/meta.yaml]: yes

**Comment by CJ-Wright** [comment]
Looks good to me!

**Comment by CJ-Wright** [comment]
@elatt welcome to conda-forge!

**Pull request merged**